packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -296,6 +311,65 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 								}
 							) }
 						</p>
+						<div>
+							{ hasPaidPlan ? (
+								<FocusedLaunchSummaryItem readOnly={ true }>
+									<LeadingContentSide label="FIGURE OUT A WAY TO GET THE PLAN NAME" />
+									<TrailingContentSide
+										price={
+											<>
+												<Icon icon={ check } size={ 18 } />{ ' ' }
+												{ __( 'Purchased', __i18n_text_domain__ ) }{ ' ' }
+											</>
+										}
+									/>
+								</FocusedLaunchSummaryItem>
+							) : (
+								[
+									<FocusedLaunchSummaryItem
+										readOnly={ hasPaidDomain || selectedPaidDomain }
+										onClick={ () => defaultFreePlan && onSetPlan( defaultFreePlan ) }
+										isSelected={ ! ( hasPaidDomain || selectedPaidDomain ) && selectedPlan?.isFree }
+									>
+										<LeadingContentSide label={ __( 'Free plan', __i18n_text_domain__ ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Free Plan', { context: 'Short plan name on storage button'} )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-step-focused-launch)